client/routes/routes.jsx

```diff
@@ -1,73 +1,93 @@
 import React from 'react';
-import { Router, Route, browserHistory, IndexRoute } from 'react-router';
-import makeAuthed from '../components/app/imports/make-authed.js';
+import { useLayoutEffect, useState } from "react";
+import { Router, Switch, Route, Routes } from 'react-router-dom';
+import RequireAuth from '../components/app/imports/RequireAuth.js';
+import App from '../components/app/App';
 
-export const renderRoutes = () => (
-  <Router history={browserHistory}>
+import { browserHistory } from '../history';
 
-    <Route path='/' component={App}>
+{/* Custom Router: https://stackoverflow.com/a/70000286 */}
```

Comment:
I don't think the { } is necessary for this comment since it's outside of a JSX-styled area